src/quo/components/wallet/approval_label/schema.cljs

```diff
@@ -0,0 +1,14 @@
+(ns quo.components.wallet.approval-label.schema)
+
+(def ?schema
```

Comment:
The schema is so simple and it's like a documentation about how the component should be used. I remember @ulisesmac said recently he's not a fan of always having a separate schema file and I agree with him. It's different than styles namespace, the schema is an integral part of the component API and forcing devs to always jump to a separate file to read it has a downside.

**No suggestion for change @ajayesivan** Just sharing thoughts and I added a section `[2024-05-21] Should quo schemas be always defined in a separate file?` in this issue's description for a future discussion https://github.com/status-im/status-mobile/issues/18726

Reply:
Thank you @ilmotta for opening the discussion.

Yeah, IMO it could be just above of the the `(def view ...)` line.

Reply:
One nice thing with separate files is that the lookup becomes really easy to find similar examples.
I can look up schema files and some key and then use these cases, e.g customisation-color.
It's not so much friction either way though

Reply:
I'll add to that discussion instead

Reply:
Agreed @J-Son89. There are upsides to separate files. It's no big deal, just another point we can come back later on to assess.

Reply:
I also prefer the separate file approach for schemas. In my workflow, it's easier to look up schemas, especially since we don't have schemas for all components. I don't have any strong preferences. I'm happy to go with whatever the team decides.